### DISCUSSION TITLE
Discord Bots

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#1347] [Operations]
How can we leverage Discord bots to enhance the discord experience for our community?

### current state
the following bots are currently in use:
- [Apollo](https://apollo.fyi/)
  - schedules a community call every week that users can RSVP to (RSVP message sent to 'events' channel in discord)
- Open Collective bot
  - posts when open collective donations are made to the 'open-collective' channel
- Github bot
  - posts updates from Athens github repository to the 'updates' channel
- [Bookmarker](https://top.gg/bot/453939662168260612)
  - users can react to any message with the 🔖 emoji and the bot will direct message the user a link to the bookmarked message
- [Starboard](https://discordbotlist.com/bots/starboard)
  - users can react with a star emoji to a post. currently 5 star emoji reacts will send a post to the starboard channel 

### notes
- https://craig.chat/home/ for audio recording #1282

**Comment by joelhans** [comment]
I saw that the Obsidian Discord uses [Carl-bot](https://carl.gg/) to post resources. I can't quite tell if the bot posts this same comment every time it sees someone say "New to Obsidian" or some other string, or if they triggered the bot to post that manually. 

<img width="863" alt="Screen Shot 2021-06-11 at 4 44 19 PM" src="https://user-images.githubusercontent.com/1153921/121757989-70d7cb00-cad4-11eb-9ca0-4c5c0697969b.png">

Either way, maybe there's ways to collect information based on common needs/questions use a bot to answer in a reliable way.

**Comment by jefftangx** [comment]
Not a bot, but heard good things about https://www.guilded.gg/

**Reply by AestraDev** [reply]
**Just find other types of bot which can help for example**

• Carl bot for looks and roles 
• Dyno and Mee6 for Automod etc.

You just need to find helpful bots online and can be found on top.gg or more,

I hope it will help

**Reply by jsmorabito** [reply]
thank you for these tips! we used carl bot previously but it became unnecessary because everyone was so well behaved ☺️. as the community grows we may like to incorporate some of these bots that you mentioned to help with the consequences of scaling.
